src/stories/30_Monetization/DoubleClick.html

```diff
@@ -0,0 +1,353 @@
+<!--- {
+  "preview": "default"
+}--->
+
+<!--
+  ## Introduction
+  In an AMP story, you cannot put an `amp-ad` directly onto the page, instead, all
+  ads are fetched and displayed by the amp-story-auto-ads extension.
```

Comment:
nit: add code formatting for amp-story-auto-ads

Reply:
done.